apps/camera/js/lib/camera/focus.js

```diff
@@ -81,6 +81,7 @@ Focus.prototype.configureFocusModes = function(mode) {
   this.faceDetection =
     faceDetectionUserEnabled && faceDetectionSupported && mode === 'picture';
   this.mozCamera.addEventListener('focus', this.onAutoFocusStateChange);
+  this.focusState = undefined;
```

Comment:
Also, it should be "delete this.focusState" rather than "this.focusState = undefined;".